src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,27 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public enum Color {
+        GREEN,
+        RED,
+        YELLOW
+    }
+
+    private Random random = new Random();
+
+    public Color getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
+    }
+
+    public Random getRandom() {
+        return random;
+    }
+
+    public void setRandom(Random random) {
+        this.random = random;
     }
```

Comment:
I dont think we need this getter and setter (we need them in model classes but not in services)